skills/lark-event/references/lark-event-im.md

```
@@ -0,0 +1,165 @@
+全部声明 `auth_types = ["bot"]`，用 `--as bot` 或 `--as auto`（auto 会解析到 bot）。
+
+## 从 0 到 1：抓一条样本
+
+```bash
+lark-cli event consume im.message.receive_v1 --max-events 1 --timeout 60s --as bot
+# 然后到 bot 所在会话 / 群手动发一条消息
+```
+
+**预期 stderr**：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)